elm-compiler.cabal

```diff
@@ -160,11 +160,13 @@ Executable elm
         CommandLineRouter.hs
 
     other-modules:
-        Elm.Compiler.Version
+        Elm.Compiler.Version,
+        Paths_elm_compiler
 
     Build-depends:
         base >=4.2 && <5,
         directory >= 1.0 && < 2.0,
+        elm-compiler,
```

Comment:
Do you mind opening an issue with them to ask about it? If it is a bug, I'd be happy to remove that line!